src/NVelocity.Tests/Test/BaseTestCase.cs

```diff
@@ -68,7 +68,7 @@ public struct TemplateTest
 
 		static TemplateTest()
 		{
-			FILE_RESOURCE_LOADER_PATH = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, Path.Combine(ConfigurationManager.AppSettings["tests.src"], "templates"));
+			FILE_RESOURCE_LOADER_PATH = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "templates");
```

Comment:
This configuration was empty in the AppSettings so I just removed it instead of attempting to create new configurations.